frontend/packages/pipelines-plugin/integration-tests/features/pipelines/pipelines-as-code.feature

```diff
@@ -0,0 +1,83 @@
+@pipelines
+Feature: Create the repo
+              As a developer, I want to create, edit, delete and view the repositories
+        Background:
+            Given user has created or selected namespace "aut-pipelines"
+            Given user has installed pipelines as code
+        
+        @smoke @pre-condition
```

Comment:
@smoke is not required, As @pre-condition tagged scripts will execute by default